api/src/main/java/org/jenkinsci/plugins/workflow/flow/StashManager.java

```diff
@@ -25,27 +25,12 @@
 package org.jenkinsci.plugins.workflow.flow;
 
 import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
-import hudson.AbortException;
-import hudson.ExtensionList;
-import hudson.ExtensionPoint;
-import hudson.FilePath;
-import hudson.Util;
+import hudson.*;
```

Comment:
Generally prefer to avoid wildcard imports and to lexicographically sort all imports.